en-US/win10/samples/arduino-wiring/DCMotor.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+title: Arduino Wiring DC Motor Sample
+permalink: /en-US/win10/samples/arduino-wiring/DCMotor.htm
+lang: en-US
+---
+
+#Arduino Wiring DC Motor Sample
+
+DC motors are some of the basic elements of a maker's toolbox. Whether you're driving autonomous robots or spinning a cooling fan, a DC motor is a relatively simple way to accomplish your task! In this sample, we'll use pulse-width modulation (PWM) to vary the speed of a motor.
+
+##Components
+
+You will need:
+
+* 1 Windows 10 IoT Core enabled device, such as Raspberry Pi 2 or Minnowboard Max.
+* 1 PWM controller breakout board or hat. We're using the [PCA9685 16-channel 12-bit PWM controller from Adafruit](http://www.adafruit.com/product/815).
+* 1 330 Ω resistor.
+* 1 1N1407 diode.
+* 1 PNP transistor
```

Comment:
You should mention the model number of the transistor you're using. Different transistors can have different positions of the base, emitter and collector. The Fritzing can get confusing if someone has a different PNP.